src/components/search/autocomplete.component.tsx

```diff
@@ -20,25 +20,26 @@ import {
  * by the AutoComplete component
  */
 export interface IProps {
-  searchResults: POI[];
-  setLocation: (poi: POI) => void;
+  searchResults: (POI | Building)[];
+  setLocation: (location: POI | Building) => void;
   style: object;
 }
 
 /**
- * Displays the names of provided points of interest's
- * @param searchResults Array of POI's
+ * Displays the names of provided POIs or Buildings
+ * @param searchResults Array of POIs or Buildings
  * @param style Styles to reposition or resize
  * @param setLocation
  */
 const AutoComplete = ({ searchResults, style, setLocation }: IProps) => {
   return (
     <View style={StyleSheet.flatten([styles.container, { ...style }])}>
       <FlatList
+        keyExtractor={(item, index) => index.toString()}
         testID="autoCompleteFlatList"
         keyboardShouldPersistTaps="handled"
         data={searchResults}
-        renderItem={({ item }: { item: POI }) => (
+        renderItem={({ item }) => (
           <TouchableOpacity
             testID="touchableList"
             onPress={() => setLocation(item)}
```

Comment:
Should this be changed to `onPressOut` to avoid bugs on certain devices like #89 ? @TheGreatMarkus

Reply:
We could, but that issue seemed to be specific to the sliding panel, due to some kind of styling/sliding panel oddity. if its not causing issue as it is, I don't think there's a need to fix it.

Reply:
We can keep this in mind, and test it on samsung devices to see if it needs multiple presses

Reply:
Tested on S8, the bug is still present and seems to be more related to the `ToucheableOpacity` component

Reply:
Interesting. In that case, if onPressOut() works more reliably and doesn't affect behaviour, lets go ahead and chance every instance to that

Reply:
Done